modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -38,6 +43,7 @@ export interface PrebuildTransactionWithIntentOptions {
   comment?: string;
   memo?: Memo;
   tokenName?: string;
+  tokens?: TokenEnablement[];
```

Comment:
maybe rename this to `enableTokens` to be clear about its purpose

Reply:
I named the field `tokens` here to match the `BuildTokenEnablementOptions` type. If we rename this type here, we should probably rename it in `BuildTokenEnablementOptions` as well.